MREUnityRuntime/MREUnityRuntimeLib/Core/Actor.cs

```diff
@@ -155,7 +155,8 @@ internal void SynchronizeApp()
 
         internal void ApplyPatch(ActorPatch actorPatch)
         {
-            var parent = ParentId != null ? App.FindActor(ParentId.Value) : Parent;
+            var parentId = actorPatch.ParentId ?? ParentId;
+            var parent = parentId != null ? App.FindActor(parentId.Value) : Parent;
```

Comment:
Why are we patching parent in a separate function than the rest of the properties? Is this a holdover from the previous architecture?